### PR TITLE
Added some missing italian translations

### PR DESCRIPTION
**Pull request by scompo**
Hi, I've added some missing Italian translations.
Fixed also a couple of references and related links.

### CODE REVIEW

File: po/it.po
Comment (der-lyse) [inline po/it.po:8]: Please update this date.

File: po/it.po
Comment (der-lyse) [inline po/it.po:10]: Please add Leandro and yourself to the `Language-Team`, too.

File: po/it.po
Comment (der-lyse) [inline po/it.po:183]: There needs to be a space between the colon and the URL.

File: po/it.po
Comment (der-lyse) [inline po/it.po:192]: Same here: space between colon and URL.

**Review by der-lyse — CHANGES_REQUESTED** [review]
Buonasera e mille grazie!

Apart from a few things down below, there are still a lot of missing translations and several fuzzy ones. Would you mind updating them as well?

**Comment by scompo** [comment]
I've added the requested changes here: 3f38015dda13f4caa39a8d51ba2ec68aa64f123b

Sure, I'll take a look at the fuzzy ones and try to fill the empty ones.

Just a quick question, should I also add a translation for messages that have no changes from the original in English?

**Comment by der-lyse** [comment]
> I've added the requested changes here: [3f38015](https://github.com/newsboat/newsboat/commit/3f38015dda13f4caa39a8d51ba2ec68aa64f123b)

Excellent, looks good!

> Sure, I'll take a look at the fuzzy ones and try to fill the empty ones.

Nice!

> Just a quick question, should I also add a translation for messages that have no changes from the original in English?

Yes, please do so. This way it's much easier to tell whether there are untranslated messages or not. When looking through the file next time, one doesn't have to go through all of them again, just to notice, that they're actually "present". It will be obvious which are missing and which are translated – even if the translation is the exact same as the original source. Dedicated translation editors, such as `poedit` or `linguist` will show which messages are fuzzy or untranslated. They also provide easy navigation features to jump to the next missing or fuzzy translation.

File: po/it.po
Comment (Minoru) [inline po/it.po:124]: Please remove the space, it looks like it's not needed anymore.

File: po/it.po
Comment (Minoru) [inline po/it.po:246]: Just checking: is "YI" a typo here, should it be "I" instead? (I don't know Italian, so totally guessing here.)

File: po/it.po
Comment (Minoru) [inline po/it.po:1883]: I'm guessing you marked those as "fuzzy" because of "between". You did everything right here: `between` is an operator, so it shouldn't be translated. We already made a note to ourselves to make it clearer to the translators #1354.

File: po/it.po
Comment (Minoru) [inline po/it.po:1888]: Unlike `between`, these strings are just words, and can be translated. Again, I promise to fix this before the next translation period :)

**Review by Minoru — CHANGES_REQUESTED** [review]
Hey, thank you for the pull request! Looks like there are just 5 fuzzy and 1 untranslated messages left to get a complete Italian translation — good job!

@scompo, can I add you to the "translators" group? I ping the group two weeks before the release, so people have time to update the translations. The releases happen every three months; there might be point releases in between, but I don't stress about translations there. Newsboat is a volunteer project, so there's no obligation on your part to maintain the translation until the end of time — a group is just a nice way for me to ping everyone concerned.

File: po/it.po
Comment (scompo) [reply po/it.po:1883]: Yes, exactly, I wasn't quite sure about it, removing it then!

File: po/it.po
Comment (scompo) [reply po/it.po:1888]: No problem, got it, I'll translate them

File: po/it.po
Comment (scompo) [reply po/it.po:246]: Yep, bad keypress, good catch, I didn't notice it.
Fixing it.

File: po/it.po
Comment (scompo) [reply po/it.po:124]: Removing it

**Comment by scompo** [comment]
@Minoru Sure, I would gladly help you.

**Review by Minoru — APPROVED** [review]

**Comment by Minoru** [comment]
Cool! I sent you an invite just now.

There's three fuzzy and one untranslated messages left — wanna take a look? ;) Anyway, I'm ready to merge this once @der-lyse approved it.

**Review by der-lyse — APPROVED** [review]
Looks good to me.

**Comment by scompo** [comment]
I hope I haven't missed a anything else :)

**Comment by der-lyse** [comment]
The only thing left I can see, is to finally remove all the now unused translations at the bottom of the file. :-)

**Comment by Minoru** [comment]
Yup, that's all of them. Italian is now one of 8 languages that have complete translations — congrats! I'll merge once the CI checks are done.

Great first PR @scompo :tada:

**Comment by Minoru** [comment]
@der-lyse I think I'll start doing that myself before cutting a release — it's not that much work after all. I mean, I'd appreciate if translators do it themselves, but I probably won't spend another review round asking them to actually do it.

**Comment by scompo** [comment]
@der-lyse @Minoru I removed the commented out lines.

**Comment by der-lyse** [comment]
@Minoru, I fully agree. Looks like @scompo has beaten you this time, though. :-D

Let's merge it, once CI is finished.

**Pull request merged**
